### PR TITLE
Add Neoscroll.nvim

### PR DESCRIPTION
**Pull request by karb94**
Neoscroll is a very customizable smooth scrolling plugin written in lua. Could you please add it to the website?

By the way, thank you for setting up the website. Looks awesome! :smiley:

### CODE REVIEW

File: src/nvim-awesome.app/data/plugins/karb94-neoscroll.nvim.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Neoscroll",
+  "description": "A smooth scrolling neovim plugin written in lua",
+  "link": "https://github.com/karb94/neoscroll.nvim",
+  "tags": [
+    "scrolling",
+    "lua"
+  ],
+  "examples": []
```
Comment (lucax88x) [inline src/nvim-awesome.app/data/plugins/karb94-neoscroll.nvim.json]: you can put a video / image in there, I saw in your readme you have a vid, put in there if you want

File: src/nvim-awesome.app/data/plugins/karb94-neoscroll.nvim.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Neoscroll",
+  "description": "A smooth scrolling neovim plugin written in lua",
+  "link": "https://github.com/karb94/neoscroll.nvim",
+  "tags": [
+    "scrolling",
+    "lua"
```
Comment (lucax88x) [inline src/nvim-awesome.app/data/plugins/karb94-neoscroll.nvim.json]: no need to add lua in there, scrolling should be enough!

**Comment by lucax88x** [comment]
Thanks man! I hope it will be used :)

File: src/nvim-awesome.app/data/plugins/karb94-neoscroll.nvim.json (hunk 2 above)
Comment (karb94) [reply src/nvim-awesome.app/data/plugins/karb94-neoscroll.nvim.json]: Then you should probably remove the "lua" tag from [template.json](https://github.com/lucax88x/nvim-awesome/blob/main/data/plugins/template.json) or other people will add it too.

Comment (lucax88x) [reply src/nvim-awesome.app/data/plugins/karb94-neoscroll.nvim.json]: I totally agree!

Comment (lucax88x) [reply src/nvim-awesome.app/data/plugins/karb94-neoscroll.nvim.json]: Updated the original template.

**Comment by lucax88x** [comment]
as soon as you remove the lua tag I can merge this one

**Comment by karb94** [comment]
oops! forgot to remove it. I'll do it in a bit.

File: src/nvim-awesome.app/data/plugins/karb94-neoscroll.nvim.json (hunk 2 above)
Comment (karb94) [reply src/nvim-awesome.app/data/plugins/karb94-neoscroll.nvim.json]: You should also remove it from [CONTRIBUTING.md](https://github.com/lucax88x/nvim-awesome/blob/main/data/plugins/CONTRIBUTING.md)

Comment (lucax88x) [reply src/nvim-awesome.app/data/plugins/karb94-neoscroll.nvim.json]: Right, fixed that part and removed the template. not useful there.

**Pull request merged**